runai-secrets.py

```
@@ -0,0 +1,298 @@
+        return 0
+
+    print("[runai-secrets] kubectl apply -f -")
+    sh(["kubectl", "apply", "-f", "-"], input_bytes=manifest.encode())
+
+    print("[runai-secrets] applied. Inference TLS not applied (knative-serving ns absent).")
+    print(f"[runai-secrets] CA cert         : {ca.crt}")
+    print(f"[runai-secrets] inference cert  : {inference.crt}  (apply later in {NS_KNATIVE})")
+    print()
+    print("Add to your `aicr bundle ...` invocation:")
+    print(f"  --set runaicp:global.domain={args.fqdn} \\")
+    print(f"  --set runaicp:global.customCA.enabled=true")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)